src/compile/compile.ts

```diff
@@ -59,17 +59,26 @@ function assemble(model: Model, topLevelProperties: TopLevelProperties) {
   // autoResize has to be put under autosize
   const {autoResize, ...topLevelProps} = topLevelProperties;
 
+  const encode = model.assembleParentGroupProperties();
+  let width;
```

Comment:
Cleaned up with f9dbb9e. I didn't want to specifically pull out just `fill` because there may be other encoding properties specified in the config?